plugin/pkg/admission/noderestriction/admission.go

```diff
@@ -161,7 +163,7 @@ func (c *nodePlugin) Admit(a admission.Attributes) error {
 	}
 }
 
-func (c *nodePlugin) admitPod(nodeName string, a admission.Attributes) error {
+func (p *Plugin) admitPod(nodeName string, a admission.Attributes) error {
```

Comment:
Can we add golint comment  for the admitpod() also?

Reply:
I can add it, but private functions are usually not required to have one.

Reply:
yeah, right but I guess doing same on the private function under the same file is not a big issue. What do you think?